src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java

```diff
@@ -64,6 +66,8 @@
 
     protected InetSocketAddress clientPortAddress;
     protected InetSocketAddress secureClientPortAddress;
+    protected boolean sslQuorum = false;
+    protected boolean shouldUsePortUnification = false;
```

Comment:
This variables and special handling code can be removed, for a configuration property that both available as system properties and available as part of zoo.cfg. In such case the processing code will fall back as https://github.com/apache/zookeeper/blob/master/src/java/main/org/apache/zookeeper/server/quorum/QuorumPeerConfig.java#L296 and is consistent with the existing code style for configuration processing.

Reply:
it would be nice to expose these through JMX

Reply:
@breed Do you think it would be useful to set these properties via JMX or should it be just read-only?

I think it would be also great to expose the properties via Jetty `stat` command as well. It makes it easier to figure out which instance is already SSL/Unification enabled.